### PR TITLE
#414 Add calendar icon in the navbar which links to the calendar page

### LINKED ISSUE #414: There should be a icon in the navbar that links to the calendar feature
## Expected behaviour

When a user logs in they see a navbar with a calendar icon. When the user clicks on this icon they are taken to the calendar component.

## Details and resources

The current navbar is utilizing the calendar icon for meetings feature. This ticket will need to use the calendar icon for calendar and use the people icon for meetings.

The front end will need a route at **/calendar**. And the icon should link to this route.

Screenshot 2022-11-08 113208

## Checklist

- [x] This issue is linked to the appropriate project.
- [x] This issue has appropriate labels.

### PR DESCRIPTION
**Pull request by wendyyng**
## Proposed changes

This pull request resolves #414 . It adds a calendar icon in the navbar that links to the calendar page. The existing icon to meetings has been changed to a people icon.

image

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?
- [x] Are there screenshots for UI changes?

### CODE REVIEW

**Comment by seidior** [comment]
@wendyyng @Prabhj33t Can you please merge the latest changes from the `main` branch, making sure the merge commit starts with your issue number, and then perform the following steps on your machine to see if you can replicate the issue we talked about on Discord?

```shell
docker compose down
docker compose pull
docker compose build nginx api flyway
docker compose build webapp --no-cache
docker compose up -d
```

Thanks!

**Comment by wendyyng** [comment]
> ```shell
> docker compose build webapp --no-cache
> ```

**Review by seidior — APPROVED** [review]
Thank you for addressing my issues that I brought up, and I'm glad we were able to move past that weird CalendarMonth icon issue.

I checked out the code on my development machine, recreated the Docker containers, brought the containers up, and tested it out.

The functionality seems to match the description of the changes, the UI changes render just as shown in the screenshot and no other UI changes are observed, and the code appears to conform to code style guidelines.

All automated tests pass without error, and `yarn delint` did not modify any files or show any errors. I approve of this pull request.

**Review by rrabinovitch — APPROVED** [review]
The changes seem to all align with the ticket specs, builds successfully on my machine, and the new functionality works successfully locally. Nice job addressing the icon issues that arose in the development of this PR.

File: webapp/src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Button, Grid, IconButton } from '@mui/material';
-import EventNoteIcon from '@mui/icons-material/EventNote';
 import React, { useContext } from 'react';
-
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import PeopleIcon from '@mui/icons-material/People';
```
Comment (veranika-karpava) [inline webapp/src/components/Navbar.tsx:4]: Please add a single blank line after PeopleIcon. It'll be a separator between imported things from packages and imported things that created by team.

**Review by veranika-karpava — CHANGES_REQUESTED** [review]
All functionalities work in my local machine. Great work!

File: webapp/src/components/Navbar.tsx (hunk 1 above)
Comment (wendyyng) [reply webapp/src/components/Navbar.tsx:4]: Hi Veranika, we added the blank line! :)

**Review by veranika-karpava — APPROVED** [review]
Thank you, Wendy. Everything works in my locale machine.

**Review by daveytech — CHANGES_REQUESTED** [review]
Due to this PR including an architectural change, the preferred method would be to first create an issue "Babel should be updated to reflect MUI packages" and submit the babel changes as an individual PR and then create another PR specifically for this tickets request. This helps us out in the future if we ever need to role back only one of these changes.

**Review by phoebehala — APPROVED** [review]

**Comment by seidior** [comment]
An update for everyone following this PR: I've reverted my commit so that it can be separated out into a new issue and a new PR. The relevant issue is #422.

Next steps:
- Wendy and Prabh will commit the fix to the branch for #422 and create a pull request.
- The new PR will be reviewed, approved, and merged.
- Wendy and Prabh will merge in the changes from origin/main to the branch for this issue, issue #414.
- Wendy and Prabh will take this pull request out of draft for its own separate approval.

**Comment by wendyyng** [comment]
@daveytech The PR for the Issue #422 for updating MUI icons has been merged to main, and we have merged in the changes from origin/main to the branch for this issue.

**Review by seidior — APPROVED** [review]
I checked out the latest copy of the code on my development machine, recreated the Docker containers, brought the containers up, and tested it out.

As was before, all functionality appears to work as described, all automated tests pass without error, and yarn delint did not modify any files or show any errors.

I re-approve of this pull request.

**Review by daveytech — APPROVED** [review]
Thank you for making the changes!

**Pull request merged**